static/app/components/workflowEngine/layout/edit.tsx

```diff
@@ -0,0 +1,118 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+import type {LocationDescriptor} from 'history';
+
+import {Breadcrumbs} from 'sentry/components/breadcrumbs';
+import ButtonBar from 'sentry/components/buttonBar';
+import * as Layout from 'sentry/components/layouts/thirds';
+import SentryDocumentTitle from 'sentry/components/sentryDocumentTitle';
+import {space} from 'sentry/styles/space';
+
+export interface WorkflowEngineEditLayoutProps {
+  /**
+   * The main content for this page
+   */
+  children: React.ReactNode;
+  /**
+   * Breadcrumb info for the parent page
+   */
+  parent: {
+    /**
+     * Link to the parent page, displayed as a breadcrumb
+     */
+    href: LocationDescriptor;
+    /**
+     * Title of the parent page, displayed as a breadcrumb
+     */
+    title: React.ReactNode;
+  };
+  /**
+   * Action buttons to display in the page header
+   */
+  actions?: React.ReactNode;
+
+  /**
+   * The main page title, for example "Automations" or "Rules"
+   */
+  title?: string;
+}
+
+/**
+ * Precomposed full-width layout for Automations / Rules edit pages.
+ * The `children` are expected to include `<EditLayout.Chart>` and `<EditLayout.Panel>` components.
+ */
+function EditLayout({parent, children, actions, title}: WorkflowEngineEditLayoutProps) {
+  return (
+    <Fragment>
+      <SentryDocumentTitle title={title} />
+      <Layout.Page>
+        <StyledHeader>
+          <Layout.HeaderContent>
+            <Breadcrumbs
```

Comment:
🤔 thoughts on having a `Layout.Breadcrumbs` that we can share in all of these views? that way we could dry up the position and props for this component. That'd also be consistent with things like `Layout.Header`

Reply:
Good idea, but it feels weird to add Breadcrumbs to the `Layout` module since it's not just presentational. I will refactor this to populate breadcrumbs via context to DRY this up though.

Reply:
Pushed in 0bebc4ca4d3